packages/tools/devtools/devtools-view/src/components/data-visualization/EditableValueView.tsx

```diff
@@ -5,30 +5,42 @@
 
 import React from "react";
 
-import { Dropdown, Input, InputOnChangeData, Option, tokens } from "@fluentui/react-components";
+import { Input, InputOnChangeData } from "@fluentui/react-components";
 import { EditType, FluidObjectValueNode, SendEditData } from "@fluid-experimental/devtools-core";
 import { useMessageRelay } from "../../MessageRelayContext";
+
 /**
- * Input to {@link TreeItem}
+ * Input to {@link EditableValueView}
  */
 export interface EditableValueViewProps {
 	node: FluidObjectValueNode;
 	containerKey: string;
 }
 
 /**
- * Constructs a tree element from the provided header and child contents.
- *
- * Intended to be used inside an outer {@link @fluentui/react-components/unstable#Tree} context.
+ * Constructs a editable text field to allow editing of a DDS' content.
  */
 export function EditableValueView(props: EditableValueViewProps): React.ReactElement {
 	const { node, containerKey } = props;
 	const messageRelay = useMessageRelay();
+
+	/**
+	 * value is the current value of the text field.
+	 */
 	const [value, setValue] = React.useState<string>(String(node.value));
+
+	/**
+	 * isEditing is whether or not the user is currently attempting to edit a value. If they are it will not update so their changes are not overwritten
+	 */
 	const [isEditing, setIsEditing] = React.useState<boolean>(false);
+
+	/**
+	 * isTextBox is whether or not the area should allow text or numbers only.
+	 */
 	const [isTextBox, setIsTextBox] = React.useState<boolean>(
 		node.typeMetadata === "number" || node.typeMetadata === "SharedCounter" ? false : true,
```

Comment:
Sorry, I missed this in the first review. We can't have any special casing against `typeMetadata` Specifically, we can't special case against DDS types (since the system is intended to be extensible for custom DDSs). Instead, we should probably do something like `typeof node.value === "number"` to check what type the actual data is. And then the `editProps` object should specify what type(s) the value _can be_ when edited.